Apply the changes from https://github.com/facebook/react-native/pull/33726 
so that we can run on a different port. This can be removed once that PR lands 
in react-native stable

diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.fb.js
index f37be3e..8ff1683 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.fb.js
@@ -5302,7 +5302,7 @@ var getViewConfigForType =
 // % 2 === 0 means it is a Fabric tag.
 // This means that they never overlap.
 
-var nextReactTag = 2;
+var nextReactTag = 999992;
 
 // TODO: Remove this conditional once all changes have propagated.
 if (registerEventHandler) {
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.js
index 39be910..68e9a22 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-dev.js
@@ -4810,7 +4810,7 @@ var getViewConfigForType =
 // % 2 === 0 means it is a Fabric tag.
 // This means that they never overlap.
 
-var nextReactTag = 2;
+var nextReactTag = 999992;
 
 // TODO: Remove this conditional once all changes have propagated.
 if (registerEventHandler) {
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.fb.js
index bd8208a..75c9d37 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.fb.js
@@ -1851,7 +1851,7 @@ var _nativeFabricUIManage = nativeFabricUIManager,
     _nativeFabricUIManage.unstable_getCurrentEventPriority,
   getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
-  nextReactTag = 2;
+  nextReactTag = 999992;
 registerEventHandler && registerEventHandler(dispatchEvent);
 var ReactFabricHostComponent = (function() {
   function ReactFabricHostComponent(
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.js
index 058a5e1..87e8af5 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-prod.js
@@ -1847,7 +1847,7 @@ var _nativeFabricUIManage = nativeFabricUIManager,
     _nativeFabricUIManage.unstable_getCurrentEventPriority,
   getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
-  nextReactTag = 2;
+  nextReactTag = 999992;
 registerEventHandler && registerEventHandler(dispatchEvent);
 var ReactFabricHostComponent = (function() {
   function ReactFabricHostComponent(
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.fb.js
index 88a4d7e..88b9720 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.fb.js
@@ -1972,7 +1972,7 @@ var _nativeFabricUIManage = nativeFabricUIManager,
     _nativeFabricUIManage.unstable_getCurrentEventPriority,
   getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
-  nextReactTag = 2;
+  nextReactTag = 999992;
 registerEventHandler && registerEventHandler(dispatchEvent);
 var ReactFabricHostComponent = (function() {
   function ReactFabricHostComponent(
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.js
index 791e998..740a8b3 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactFabric-profiling.js
@@ -1895,7 +1895,7 @@ var _nativeFabricUIManage = nativeFabricUIManager,
     _nativeFabricUIManage.unstable_getCurrentEventPriority,
   getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
-  nextReactTag = 2;
+  nextReactTag = 999992;
 registerEventHandler && registerEventHandler(dispatchEvent);
 var ReactFabricHostComponent = (function() {
   function ReactFabricHostComponent(
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.fb.js
index fa3df1e..7822e98 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.fb.js
@@ -5531,7 +5531,7 @@ var UPDATE_SIGNAL = {};
 // % 10 === 1 means it is a rootTag.
 // % 2 === 0 means it is a Fabric tag.
 
-var nextReactTag = 3;
+var nextReactTag = 999993;
 
 function allocateTag() {
   var tag = nextReactTag;
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
index 3772e42..041524c 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js
@@ -5024,7 +5024,7 @@ var UPDATE_SIGNAL = {};
 // % 10 === 1 means it is a rootTag.
 // % 2 === 0 means it is a Fabric tag.
 
-var nextReactTag = 3;
+var nextReactTag = 999993;
 
 function allocateTag() {
   var tag = nextReactTag;
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.fb.js
index 34c20b4..c764e34 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.fb.js
@@ -1912,7 +1912,7 @@ function shim$1() {
 var getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
   UPDATE_SIGNAL = {},
-  nextReactTag = 3;
+  nextReactTag = 999993;
 function allocateTag() {
   var tag = nextReactTag;
   1 === tag % 10 && (tag += 2);
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
index 3fb794d..f3a6a5b 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js
@@ -1906,7 +1906,7 @@ function shim() {
 var getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
   UPDATE_SIGNAL = {},
-  nextReactTag = 3;
+  nextReactTag = 999993;
 function allocateTag() {
   var tag = nextReactTag;
   1 === tag % 10 && (tag += 2);
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.fb.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.fb.js
index f9734c0..847f3a3 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.fb.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.fb.js
@@ -2033,7 +2033,7 @@ function shim$1() {
 var getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
   UPDATE_SIGNAL = {},
-  nextReactTag = 3;
+  nextReactTag = 999993;
 function allocateTag() {
   var tag = nextReactTag;
   1 === tag % 10 && (tag += 2);
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.js b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.js
index b875b0e..7441160 100644
--- a/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.js
+++ b/node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling.js
@@ -1954,7 +1954,7 @@ function shim() {
 var getViewConfigForType =
     ReactNativePrivateInterface.ReactNativeViewConfigRegistry.get,
   UPDATE_SIGNAL = {},
-  nextReactTag = 3;
+  nextReactTag = 999993;
 function allocateTag() {
   var tag = nextReactTag;
   1 === tag % 10 && (tag += 2);
diff --git a/node_modules/react-native/React-Core.podspec b/node_modules/react-native/React-Core.podspec
index 375bd9e..1721fef 100644
--- a/node_modules/react-native/React-Core.podspec
+++ b/node_modules/react-native/React-Core.podspec
@@ -47,7 +47,7 @@ Pod::Spec.new do |s|
   s.compiler_flags         = folly_compiler_flags + ' ' + boost_compiler_flags
   s.header_dir             = "React"
   s.framework              = "JavaScriptCore"
-  s.pod_target_xcconfig    = { "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/RCT-Folly\" \"${PODS_ROOT}/Headers/Public/React-hermes\" \"${PODS_ROOT}/Headers/Public/hermes-engine\" \"${PODS_ROOT}/Headers/Public/FlipperKit\" \"$(PODS_ROOT)/Headers/Public/ReactCommon\" \"$(PODS_ROOT)/Headers/Public/React-RCTFabric\"", "DEFINES_MODULE" => "YES" }
+  s.pod_target_xcconfig    = { "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/RCT-Folly\" \"${PODS_ROOT}/Headers/Public/React-hermes\" \"${PODS_ROOT}/Headers/Public/hermes-engine\" \"${PODS_ROOT}/Headers/Public/FlipperKit\" \"$(PODS_ROOT)/Headers/Public/ReactCommon\" \"$(PODS_ROOT)/Headers/Public/React-RCTFabric\"", "DEFINES_MODULE" => "YES", "GCC_PREPROCESSOR_DEFINITIONS" => "RCT_METRO_PORT=${RCT_METRO_PORT}" }
   s.user_target_xcconfig   = { "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/Headers/Private/React-Core\""}
   s.default_subspec        = "Default"
 
